src/styles/index.scss

```diff
@@ -1,3 +1,86 @@
+$watch-size: 80vmin;
+$minute-hand-length: 20vmin;
+$minute-hand-width: 3vmin;
+$second-hand-length: 38vmin;
+$second-hand-width: 1.5vmin;
+$center-circle-diameter: 5vmin;
```

Comment:
tip for next projects: generally separate your css files (one for every block, one for variables, one just for imports, one for global code)